scripts/README.md

```diff
@@ -0,0 +1,217 @@
+
+### [build.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/.sh)
+
+Script used by CI jobs to wrap xcodebuild invocations with options.
+
+### [build_non_firebase_sdks.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh)
+
+CI script to build binary versions of non-Firebase SDKs for QuickStart testing.
+
+### [build_zip.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build_zip.sh)
+
+CI script for building the zip distribution.
+
+### [buildcache.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/buildcache.sh)
+
+Clang options for the buildcache GitHub action.
+
+### [change_headers.swift](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/change_headers.swift)
+
+Utility script to update source to repo-relative headers.
+
+### [check_secrets.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check_secrets.sh)
+
+CI script to test if secrets are available (not running on a fork).
+
+### [collect_metrics.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/collect_metrics.sh)
+
+CI script to collects project health metrics and uploads them to a database.
+
+### [configure_test_keychain.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/configure_test_keychain.sh)
+
+CI script to setup the keychain for macOS and Catalyst testing.
+
+### [cpplint.py](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/cpplint.py)
+
+Firestore script for C++ linting.
+
+### [create_pull_request.rb](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/create_pull_request.rb)
+
+Utility used by CI scripts to create issues and PRs.
+
+### [decrypt_gha_secret.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/decrypt_gha_secret.sh)
+
+CI script to decrypt a GitHub Actions secret.
+
+### [encrypt_gha_secret.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/encrypt_gha_secret.sh)
+
+CI script to encrypt a GitHub Actions secret.
+
+### [fuzzing_ci.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/fuzzing_ci.sh)
+
+Firestore CI script to run fuzz testing.
+
+### [generate_access_token.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/generate_access_token.sh)
+
+Script to generate a Firebase access token used by Remote config integration tests.
+
+### [install_prereqs.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scriptsinstall_prereqs.sh)
+
+Utility CI script to provide configuration for build.sh
+
+### [localize_podfile.swift](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/localize_podfile.swift)
+
+Utility script to update a Podfile to point to local podspecs.
+
+### [make_release_notes.py](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/make_release_notes.py)
+
+Converts GitHub flavored markdown changelogs to release notes.
```

Comment:
nit: maybe say `Converts GitHub-flavored markdown changelogs to devsite-compatible release notes.` to be more specific

Reply:
done